### PR TITLE
AOAF-414-Zip-portal-Build-header-footer-and-back-button-welsh-equivalent

### PR DESCRIPTION
**Pull request by okoye-companieshouse**

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/en/global.json [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+    "feedback": "feedback",
+    "new_service_postfix": "will help us to improve it.",
+    "back": "Back",
+    "sign_out": "Sign out",
+    "policies": "Policies",
+    "cookies": "Cookies",
+    "contact_us": "Contact us",
+    "accessibility_statement": "Accessibility statement",
+    "developers": "Developers",
+    "content_info_prefix": "All content is available under the",
```
[the remark this answers is not in the document]
Comment (okoye-companieshouse) [reply locales/en/global.json:15]: Will use pre_text and post_text then as they are not really links

[the remark this answers is not in the document]
Comment (okoye-companieshouse) [reply locales/en/global.json:15]: Thanks

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**